source/_integrations/zha.markdown

```diff
@@ -610,6 +610,13 @@ The visualization shows multi-hop connections between your paired devices and th
 
 The exact method in which these values are reported depends on the Zigbee network stack used on each device. LQI values can be modified at each step as the message propagates through the mesh networking matrix.
 
+#### Why some links are missing in Zigbee network topology maps
+
+Missing links between Zigbee End Devices (i.e., battery-powered devices) in the Zigbee network visualization map are usually nothing to worry about. This is a common occurrence and is generally not a problem or fault if the device is still reporting state changes. This happens with sleepy Zigbee End Devices and does not mean that the device is no longer connected.
+
+The reason for this is that some end devices (e.g., Xiaomi door sensors) sleep for an extended period, causing the parent Zigbee Router to remove them from its child table via a feature called router child ageing. Since using child ageing and removing it from the child table is a Zigbee 3.0 feature, this does not always occur because not all Zigbee Router devices use child ageing.
```

Comment:
```suggestion
Some end devices (for example, Xiaomi door sensors) sleep for an extended period, causing the parent Zigbee Router to remove them from its child table via a feature called router child aging. Since using child aging and removing them from the child table is a Zigbee 3.0 feature, this does not always occur because not all Zigbee router devices use child aging.
```